src/server/campaignd/server.cpp

```diff
@@ -347,13 +347,17 @@ void server::load_config()
 	// One month probably will be fine (#TODO: testing needed)
 	update_pack_lifespan_ = cfg_["update_pack_lifespan"].to_time_t(30 * 24 * 60 * 60);
 
-	if(const auto& svinfo_cfg = server_info()) {
-		server_id_ = svinfo_cfg["id"].str();
-		feedback_url_format_ = svinfo_cfg["feedback_url_format"].str();
-		web_url_ = svinfo_cfg["web_url"].str(default_web_url);
-		license_notice_ = svinfo_cfg["license_notice"].str(default_license_notice);
+	config svinfo_cfg;
+	// If the config file isn't empty, copy the server info tag instead of using all defaults
+	if(const auto& cfg = server_info()) {
```

Comment:
Could also change `server_info` to use `child_or_empty` and assign unconditionally.

Reply:
Just pushed a version that does just this